[Bug][Compile] clang compile fail on vparquet_scanner

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.


### Version

master

### What's Wrong?

```cpp
/home/bitetheddddt/dbdev/incubator-doris/be/src/vec/exec/vparquet_scanner.h:52:12: error: 'doris::vectorized::VParquetScanner::get_next' hides overloaded virtual function [-Werror,-Woverloaded-virtual]
    Status get_next(Block* block, bool* eof);
           ^
/home/bitetheddddt/dbdev/incubator-doris/be/src/exec/parquet_scanner.h:63:20: note: hidden overloaded virtual function 'doris::ParquetScanner::get_next' declared here: different number of parameters (4 vs 2)
    virtual Status get_next(Tuple* tuple, MemPool* tuple_pool, bool* eof, bool* fill_tuple);
                   ^
1 error generated.
[27/671] Building CXX object src/exec/CMakeFiles/Exec.dir/csv_scan_node.cpp.o
ninja: build stopped: subcommand failed.
```

### What You Expected?

fix it

### How to Reproduce?

_No response_

### Anything Else?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/be/src/exec/parquet_scanner.h
+++ b/be/src/exec/parquet_scanner.h
@@ -54,16 +54,20 @@ public:
                    const std::vector<TNetworkAddress>& broker_addresses,
                    const std::vector<TExpr>& pre_filter_texprs, ScannerCounter* counter);
 
-    ~ParquetScanner();
+    ~ParquetScanner() override;
 
     // Open this scanner, will initialize information need to
-    virtual Status open();
+    Status open() override;
 
     // Get next tuple
-    virtual Status get_next(Tuple* tuple, MemPool* tuple_pool, bool* eof, bool* fill_tuple);
+    Status get_next(Tuple* tuple, MemPool* tuple_pool, bool* eof, bool* fill_tuple) override;
+
+    Status get_next(vectorized::Block* block, bool* eof) override {
+        return Status::NotSupported("Not Implemented get block");
+    }
 
     // Close this scanner
-    virtual void close();
+    void close() override;
 
 protected:
     // Read next buffer from reader

--- a/be/src/vec/exec/vparquet_scanner.h
+++ b/be/src/vec/exec/vparquet_scanner.h
@@ -44,12 +44,12 @@ public:
                     const std::vector<TNetworkAddress>& broker_addresses,
                     const std::vector<TExpr>& pre_filter_texprs, ScannerCounter* counter);
 
-    virtual ~VParquetScanner();
+    ~VParquetScanner() override;
 
     // Open this scanner, will initialize information need to
-    Status open();
+    Status open() override;
 
-    Status get_next(Block* block, bool* eof);
+    Status get_next(Block* block, bool* eof) override;
 
 private:
     Status _next_arrow_batch();

